### PR TITLE
Search sighting front

### PR DESCRIPTION
**Pull request by callumfg**
Search sighting frontend

### CODE REVIEW

File: ClientApp/src/Api/apiClient.ts
Comment (chloejonessoftwire) [inline ClientApp/src/Api/apiClient.ts:52]: quite a lot of empty lines here

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.tsx:45]: again, empty space

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.tsx]: lots of empty space :)

**Review by chloejonessoftwire — COMMENTED** [review]
sorry for being pedantic, there are quite a few empty lines that you might want to delete to neated up the code. otherwise looks good ! :D

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx
Comment (littlemauli) [reply ClientApp/src/Pages/SearchSightings/SearchSightings.tsx]: I agree with Chloe on the empty spaces

**Review by littlemauli — COMMENTED** [review]
it is so neat! just a few empty spaces.

File: ClientApp/src/Pages/SearchSightings/SearchSightings.scss
Comment (ChristianMcc) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.scss:10]: all these styles are a bit indented I think

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx
Comment (ChristianMcc) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.tsx:162]: some spare white space

**Review by ChristianMcc — APPROVED** [review]
Looks great! couple of formatting tweaks - have approved so you can merge when those are done without another review :)

**Pull request merged**
